examples/ssh/mysql.js

```diff
@@ -1,34 +1,85 @@
-var _var = D.device.createVariable;
+/**
+ * Domotz Custom Driver 
+ * Name: Mysql Server Monitoring
+ * Description: Monitors Mysql service on Linux. 
+ * 
+ * Communication protocol is SSH.
+ * 
+ * Tested on Linux:
+ * Ubuntu 22.04
+ * Debian 10
+ * Centos 7
+ * 
+ * Please Note that:
+ *  
+ * - Your MySQL must run as systemd service for this command to work
+ * - This works in Debian derivatives and Centos, other Linux distributions might different paths
+ * 
+ * 
+ * The driver will create the following variables:
+ * 
+ * - Status
+ * - Version
+ * - Used Ram
+ * - Uptime
+ * - Threads
+ * - Questions
+ * - Opens
+ * - Flush tables
+ * - Open tables
+ * - Average Queries
+ * - Errors
+ * 
+**/
+
+
+// Mysql options, set here username and password to be used
+var mysqlUser=""
```

Comment:
The driver crashes when its out of the box.

Have you tested on non-protected mysql servers as well?